include/chemfiles/Connectivity.hpp

```diff
@@ -322,6 +342,8 @@ class Connectivity final {
     mutable sorted_set<Improper> impropers_;
     /// Is the cached content up to date ?
     mutable bool uptodate_ = false;
+    /// Store the bond orders
+    std::vector<BondOrder> bondorders_;
```

Comment:
It feels like `bond_orders_` would be more readable, but this is highly subjective, so do as you want =)

Reply:
readable like `uptodate_` :smile: ? I was copying your style, so that's why I clubbed it into one word!

Reply:
One point for you =). We can change `uptodate_` to `up_to_date_` too (or to a better name if you can think of one!)